client/src/components/PropertiesMap.tsx

```diff
@@ -112,13 +193,88 @@ export default class PropertiesMap extends Component<Props, State> {
   }
 
   componentDidMount() {
+    const { newAssocAddrs, newAddrsBounds } = this.getAddrsPointsAndBounds();
+
+    // sets things up, including initial portfolio level map view
+    this.setState(
+      {
+        addrsBounds: newAddrsBounds,
+        addrsPoints: newAssocAddrs,
+      },
+      () => {
+        // yeah, this sucks, but it seems to be more consistent with
```

Comment:
lol the dev internal monologue 😆
i do think the reasoning behind this approach is valuable if you want to keep the comment (or some version of it) in

Reply:
haha yeah this was i think from sam, or maybe atul. i also found it useful context - i'll leave it